sdk/include/psdk/ntgdi.h

```diff
@@ -151,7 +151,7 @@ W32KAPI
 BOOL
 APIENTRY
 NtGdiGetFontResourceInfoInternalW(
-    _In_reads_z_(cwc) LPWSTR pwszFiles,
+    _In_reads_z_(cwc) LPCWSTR pwszFiles,
```

Comment:
```suggestion
    _In_reads_z_(cwc) PCWSTR pwszFiles,
```
or PCWCH if it's a non-NUL-terminated string.

Reply:
```suggestion
    _In_reads_z_(cwc) PCWCH pwszFiles,
```